packages/api/src/hooks/useLoginHistory.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import moment from 'moment';
+import Bowser from 'bowser';
+import { LoginHistory as TLoginHistory } from '@deriv/api-types';
+import { Localize } from '../../../translations/src/i18next/i18next';
+import { TData } from '../../types';
+import useFetch from '../useFetch';
+
+const API_FETCH_LIMIT = 50;
+
+type TUserAgent =
+    | (Bowser.Parser.Details & {
+          app?: string;
+      })
+    | undefined;
+
+const getFormattedData = (login_history: TLoginHistory) => {
```

Comment:
Why does this have to be here ? This can be moved to utils package. Plus we have localize and such in api/hooks is not a good idea in my opinion. Whole lot of business logic we are putting at the top level.

Reply:
can refer to `useAccountsList` on the data transformation

Reply:
can also refer to `unFormatLocaleString`